guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -13,19 +13,27 @@ To install {SmartProxyServer}, complete the following steps:
 ----
 # {package-update}
 ----
-ifeval::["{build}" == "satellite"]
```

Comment:
I'd suggest we either drop this file all together and use https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_installing-the-satellite-server-packages.adoc or create a similarly structure file for smart-proxy/capsule. As there is a lot of nuance missing here per OS.